CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+v1.3.0
+
+15-11-2017
+
+* Spellcheck will now always initialise with options from the CLI

```

Comment:
As discussed, it looks to me like only the spell and spellFile functions are affected, which aren't used by the CLI.